client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -126,7 +126,7 @@
          * Autosave an item
          */
         this.save = function saveAutosave(item) {
-            if (item._editable && item._locked) {
+            if (item._editable && !item._locked) {
```

Comment:
Autosave works when an item is edited but when an item it will be locked.
So why is this change required?